data/campaigns/The_Rise_Of_Wesnoth/scenarios/08_Clearwater_Port.cfg

```diff
@@ -731,33 +731,35 @@
 
     #The victory
 
+#define BOARDING_EVENT SHIP_EVENT ABSCISSA ORDINATE BONUS
+    # Refactor repeated boarding events
     [event]
         name=moveto
         first_time_only=no
         [filter]
             side=1
-            x=31
-            y=16
+            x={ABSCISSA}
+            y={ORDINATE}
         [/filter]
 
         [if]
             [have_unit]
                 id=Prince Haldric
-                x=31
-                y=16
+                x={ABSCISSA}
+                y={ORDINATE}
             [/have_unit]
 
             [then]
                 [if]
                     [variable]
-                        name=First_Ship
+                        name={SHIP_EVENT}
```

Comment:
I think the `First_Ship`, `Second_Ship` and `Third_Ship` variables are old code that could be replaced by using `[remove_event]` when a ship departs.

Reply:
I don't understand what that means. All I'm trying to do here is remove the duplicate code, not change existing behaviour, although if there are deprecated patterns that we should be avoiding I can try to fix that, although remember I'm still a WML novice.

Reply:
Ah, sorry I assumed that you weren't a WML novice. I think I'd need to try writing the whole thing myself in order to explain what I mean.

Reply:
If you can improve the event itself, go ahead and add your changes on top of this. The intent was simply to satisfy #6298: three similar events that (apparently unintentionally) introduces an inconsistency between whether Commander Aethyr speaks, or the narrator. Removing the unnecessary duplication helps keeps the events consistent, that's all my intention was. Any further improvement on top of that is just a bonus and separate to this PR.